nautobot/extras/jobs.py

```diff
@@ -1185,37 +1151,35 @@ def _run_job():
                     if output:
                         job.results["output"] += "\n" + str(output)
 
-                if job.failed:
-                    job.logger.warning("job failed")
-                    job_result.set_status(JobResultStatusChoices.STATUS_FAILED)
-                else:
-                    job.logger.info("job completed successfully")
-                    job_result.set_status(JobResultStatusChoices.STATUS_COMPLETED)
+                job.logger.info("job completed successfully")
 
                 if not commit:
-                    raise AbortTransaction()
+                    raise AbortTransaction("Database changes have been reverted automatically.")
 
         except AbortTransaction:
             if not job_model.read_only:
                 job.log_info(message="Database changes have been reverted automatically.")
 
-        except Exception as exc:
-            stacktrace = traceback.format_exc()
-            job.log_failure(message=f"An exception occurred: `{type(exc).__name__}: {exc}`\n```\n{stacktrace}\n```")
```

Comment:
How do exceptions thrown during job execution get recorded now?

Reply:
They get stored on the `JobResult.traceback` field and unless otherwise handled by decorating the task with a `throws=` argument, they result in the job status of `FAILURE`.

![Cursor_and_Job_Result__File_Upload_Failure_-_Nautobot](https://user-images.githubusercontent.com/138052/216102849-b835c78a-35ee-4367-a977-0dc3e2d4c8b6.png)

Reply:
Any traceback will result in a failure unless it's defined as a known exception type such as using [`autoretry_for`](https://docs.celeryq.dev/en/stable/userguide/tasks.html?highlight=exceptions#automatic-retry-for-known-exceptions)

Also traceback is currently part of the "Additional Data" tab:

![image](https://user-images.githubusercontent.com/138052/216429210-343a0cf1-6d3b-4796-b104-852b8daed180.png)

Unless you're thinking of showing it in the job log as well?